pkg/azuredisk/controllerserver.go

```diff
@@ -382,8 +382,75 @@ func (d *Driver) ControllerGetVolume(context.Context, *csi.ControllerGetVolumeRe
 }
 
 // ControllerModifyVolume modify volume
-func (d *Driver) ControllerModifyVolume(context.Context, *csi.ControllerModifyVolumeRequest) (*csi.ControllerModifyVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+func (d *Driver) ControllerModifyVolume(ctx context.Context, req *csi.ControllerModifyVolumeRequest) (*csi.ControllerModifyVolumeResponse, error) {
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume ID missing in the request")
+	}
+
+	if err := d.ValidateControllerServiceRequest(csi.ControllerServiceCapability_RPC_MODIFY_VOLUME); err != nil {
+		return nil, status.Errorf(codes.Internal, "invalid modify volume req: %v", req)
+	}
+	diskURI := volumeID
+
+	if err := azureutils.IsValidDiskURI(diskURI); err != nil {
```

Comment:
not necessary here since since `azureutils.GetDiskName(diskURI)` already checks whether diskURI is valid

Reply:
remove it